source/1_engine/rex_engine/include/rex_engine/filesystem/path.h

```diff
@@ -216,6 +187,54 @@ namespace rex
       rsl::string_view root;
       rsl::string_view tail;
     };
+
+    // These functions are required to be implemented by platform specific code
```

Comment:
Could you make the difference between platform code and generic code a bit more clear.
This line of comment is easily overlooked

Reply:
done